doc/Makefile

```diff
@@ -17,6 +17,9 @@ CHPL2RSTOPTS = --output=rst --prefix=$(CHPL_DOC_PRIMERS_DIR) --link=$(WEB_DOC_DI
 CHPL_DOC_HELLO_DIR = $(SOURCEDIR)/examples/hellos
 CHPL2RSTOPTS_HELLO = --output=rst --prefix=$(CHPL_DOC_HELLO_DIR) --link=$(WEB_DOC_DIR)$(WEB_DOC_VERSION)
 
+SPECSRCDIR = $(SOURCEDIR)/language/spec
+GETSYNTAX = ../util/config/run-in-venv.bash ./util/collect-syntax.py
```

Comment:
Something needs to change here after PR 16694 - I recommend you rebase now that PR is merged. However I don't see any reason why collect-syntax.py needs to run in the venv.